### ISSUE TITLE
Make font customization more flexible

### ISSUE THREAD

**Issue by utkuufuk** [theme-next/hexo-theme-next#249]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

Hi there,

Currently the font options selected for the `headings` field customizes both the section headers inside a post and post titles.

I think it would be great if it was possible to define 2 sets of options, one for section headings and one for post titles (along with the other options like `logo` and `codes`).

Then the `_config.yml` of the theme would look like below:
```yml
font:
  enable: true

  headings:
    external: true
    family: Acme
    size: 18

  title:
    external: true
    family: Russo One
    size: 20
```

**Comment by stevenjoezhang** [comment]
Added to NexT v7.x Roadmap

**Comment by stevenjoezhang** [comment]
Fixed in #1005, #1006
